config/locales/models/users/en.yml

```diff
@@ -13,6 +13,7 @@ en:
         full_name: Name
         grace_credits: Grace Credits
         id_number: ID number
+        inactive: inactive
```

Comment:
This is a bit awkward, but the keys here should match the actual attribute names, so in this case it would be `hidden` instead of `inactive` as the _key_. The text can stay as "inactive" though, since that's the text we display to the user